source/client/ui/PropertyOptions.ts

```diff
@@ -120,22 +124,47 @@ export default class PropertyOptions extends CustomElement
         let buttons;
         if (indexMap) {
             buttons = indexMap.map(index =>
-                html`<ff-button role="radio" aria-checked=${index === value ? true : false} tabbingIndex=${indexMap.includes(value) ? (index === value ? 0 : -1) : (index === indexMap[0] ? 0 : -1)} .text=${language ? language.getLocalizedString(options[index]) : options[index]} .index=${index} .selectedIndex=${value} @click=${this.onButtonClick}>
```

Comment:
I'll have to test but I believe we still need aria-checked for screen reader support.

Reply:
Fixed in 135829e (will squash it together once all discussions are resolved)